app/controllers/orcid_works_controller.rb

```diff
@@ -0,0 +1,27 @@
+class OrcidWorksController < UserController
+  before_action :authenticate!
+
+  def create
+    authorship = Authorship.find(params[:authorship_id])
```

Comment:
Here we're probably going to want to scope this query to the current user's authorships.